_community/google-groups.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Google Groups - Managing List Membership
+layout: text-optimized-width
+
+---
+
+# {{ page.title }}
+
+This documents how to join and leave a Google Group via webpage and email.
```

Comment:
How about? “The OSG Consortium is moving to using Google Groups for managing mailing lists that end in `osg-htc.org`. This page describes how to manage your membership in these lists.”